src/Domains/domain.jl

```diff
@@ -133,49 +133,55 @@ function PeriodicPlane(
 end
 
 """
-    Base.ndims(::Column)
-    
-Query function for `Column` domain (number of dimensions)
+    struct HVSpace2D{FT} <: AbstractHybridDomain 
 """
-Base.ndims(::Column) = 1
+struct HVSpace2D{FT} <: AbstractHybridDomain{FT}
+    xlim::Tuple{FT, FT}
+    zlim::Tuple{FT, FT}
+    helement::Int32
```

Comment:
Let's turn `hlement` and `velement` into a Tuple with name `nelements` to follow the interfaces of the other domains.